plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/utils/PlayAssetDeliveryUtils.java

```diff
@@ -46,6 +48,20 @@ public static Dictionary constructAssetPackStateDictionary(
     return returnDict;
   }
 
+  public static Dictionary constructAssetPackStatesDictionary(long totalBytes) {
```

Comment:
Could we change this method to take a dictionary as the second argument, rather than using appendToAssetPackStatesDictionary? That would be more consistent with the other methods.

Reply:
I modified this function so that it can take a Dictionary constructed by stream. I also decided to keep appendToAssetPackStatesDictionary() since it is used as a helper function for testcases.